ui/pages/send/send-content/send-content.component.js

```diff
@@ -84,24 +88,52 @@ export default class SendContent extends Component {
     );
   }
 
+  checkContractAddress = () => {
+    const { recipient, tokenAddressList } = this.props;
+
+    getSymbolAndDecimals(recipient.userInput, tokenAddressList).then(
+      (result) => {
+        if (
+          (result.symbol !== undefined && result.symbol !== '') ||
+          (result.decimals !== undefined && result.decimals !== '')
+        ) {
+          this.setState({ isContractAddress: true });
+        }
+      },
+    );
+  };
+
   maybeRenderAddContact() {
     const { t } = this.context;
-    const { isOwnedAccount, contact = {}, to } = this.props;
-    const { showNicknamePopovers } = this.state;
+    const { isOwnedAccount, contact = {}, to, recipient } = this.props;
+    const { showNicknamePopovers, isContractAddress } = this.state;
+
+    this.checkContractAddress();
```

Comment:
any reason this shouldn't just be called in the `render` method?

Reply:
Fixed in: ad39793344c75085bf0a75435319383fa7f7b514